pkg/engine/jmespath/functions_test.go

```diff
@@ -504,3 +504,25 @@ func Test_Base64Decode_Secret(t *testing.T) {
 	assert.Assert(t, ok)
 	assert.Equal(t, string(result), "Hello, world!")
 }
+
+func Test_TimeSince(t *testing.T) {
```

Comment:
Please implement a table test for the multiple different testcases - See `Test_LabelMatch` in this file for reference.